src/models/searchservice/response/QueryRulesActionsData.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Data returned from the Answers query rules system.
+ *
+ * @public
+ */
+export interface QueryRulesActionsData {
+  /**
+   * The unique identifier for this query rule.
+   */
+  key: string,
+  /**
+   * The data returned from the query rule.
+   */
+  data?: Record<string, unknown>,
+  /**
+   * Any errors returned from the query rule.
+   */
+  errors?: string
```

Comment:
Do we know if this can return multiple errors? In general Rose prefers us to stick with the same name as the API which would mean changing this to a singular 'error'

Reply:
Yes, there can be multiple errors. I've updated the error object type after getting more clarification from the backend on what will be returned.